zingo-account/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+//! A micro crate to provide a wrapper around the zcash_client_backend account trait.
+//! By constraining the wrapper to be independent of internal zingolib functionality
+//! we expose a shareable component, and refine the definition of zingolib.
+
+/// The a view-only abstraction that provides a management interface for
```

Comment:
"The a view-only" is a typo? also comment seems to just ...end. I am ok to chalk that up to the TODO if you want